### PR TITLE
doc/rados: edit change-mon-elections

### PR DESCRIPTION
**Pull request by zdover23**
Edit doc/rados/operations/change-mon-elections.rst.

https://tracker.ceph.com/issues/58485

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/operations/change-mon-elections.rst [hunk 1]
```diff
@@ -1,88 +1,100 @@
...
-The disallow Mode
-=================
-This mode lets you mark monitors as disallowed, in which case they will
-participate in the quorum and serve clients, but cannot be elected leader. You
-may wish to use this if you have some monitors which are known to be far away
-from clients.
-You can disallow a leader by running:
+The modes other than ``classic`` provide extra features. We recommend staying
+in ``classic`` mode if you don't need these extra features because it is the
+simplest mode.
```
Comment (anthonyeleven) [inline doc/rados/operations/change-mon-elections.rst:27]: Suggest s/extra/specific/ so the uncareful reader is perhaps less inclined to disregard advice.

File: doc/rados/operations/change-mon-elections.rst [hunk 2]
```diff
@@ -1,88 +1,100 @@
...
+Scores for an individual connection range from ``0`` to ``1`` inclusive and
+include whether the connection is considered alive or dead (as determined by
+whether it returned its latest ping before timeout).
 
-While this would be an unexpected occurrence, if for some reason you experience
-problems and troubleshooting makes you think your scores have become invalid,
-you can forget history and reset them by running:
+Connectivity scores are expected to remain valid. However, if during
+troubleshooting you determine that these scores have for some reason become
+invalid, forget the history and reset the scores by running a command of the
```
Comment (anthonyeleven) [inline doc/rados/operations/change-mon-elections.rst:89]: Suggest s/forget/drop/ but this is marginal.

File: doc/rados/operations/change-mon-elections.rst [hunk 3]
```diff
@@ -1,88 +1,100 @@
...
    ceph daemon mon.{name} connection scores reset
 
-While resetting scores has low risk (monitors will still quickly determine
-if a connection is alive or dead, and trend back to the previous scores if they
-were accurate!), it should also not be needed and is not recommended unless
-requested by your support team or a developer.
+Resetting connectivity scores carries little risk: monitors will still quickly
+determine whether a connection is alive or dead and trend back to the previous
+scores if those scores were accurate. Nevertheless, resetting scores ought to
+be unnecessary and it is not recommended unless requested by your support team
```
Comment (anthonyeleven) [inline doc/rados/operations/change-mon-elections.rst:99]: s/requested/advised/, again marginal.

**Review by anthonyeleven — APPROVED** [review]

File: doc/rados/operations/change-mon-elections.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/operations/change-mon-elections.rst:27]: I think all three of these suggestions are clear improvements. They're all accepted.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/51998 - Reef backport
https://github.com/ceph/ceph/pull/51999 - Quincy backport

**Pull request merged**